extension/src/shared/reearth/layers/polygon.ts

```diff
@@ -0,0 +1,43 @@
+import { FC, useMemo } from "react";
+
+import { useLayer } from "../hooks";
+import { Data, LayerAppearanceTypes } from "../types";
+
+export type PolygonAppearances = Partial<Pick<LayerAppearanceTypes, "polygon">>;
+
+export type PolygonProps = {
+  onLoad?: (layerId: string) => void;
+  visible?: boolean;
+  appearances?: PolygonAppearances;
+};
+
+export const PolygonLayer: FC<PolygonProps> = ({ onLoad, visible, appearances }) => {
+  const mergedAppearances: PolygonAppearances | undefined = useMemo(
+    () => ({
+      ...appearances,
+      polygon: {
+        ...(appearances?.polygon ?? {}),
+        hideIndicator: true,
+      },
+    }),
+    [appearances],
+  );
+
+  const url = "https://api.plateau.reearth.io/govpolygon/plateaugovs.geojson";
```

Comment:
Could you update to use PLATEAU_API_URL? This variable is come from widget setting.
```suggestion
  const url = `${PLATEAU_API_URL}/govpolygon/plateaugovs.geojson`;
```